server/controllers/componentsController.test.ts

```diff
@@ -1,42 +1,8 @@
-import { UserService } from '../services'
-import componentsController from './componentsController'
+import componentsController, { FooterViewModel, HeaderViewModel } from './componentsController'
```

Comment:
The componentsController is now much simpler since it can rely on the user object:
* being consistent, regardless of whether it's via `component` or `develop` route
* containing case load and service data

So the test can be made simpler now...